### PR TITLE
Fix docs/spin1_api Makefile

### PR DESCRIPTION
**Pull request by lplana**
updated docs/spin1_api Makefile to solve a missing reference warning.

### CODE REVIEW

File: docs/spin1_api/Makefile [hunk 1]
```diff
@@ -14,8 +14,8 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 spin1_api.pdf: spin1_api.tex threads.eps
-	latex spin1_api.tex
-	dvipdf spin1_api.dvi
+	pdflatex spin1_api.tex
+	pdflatex spin1_api.tex
```
Comment (rowleya) [inline docs/spin1_api/Makefile:18]: Did you mean to put this call in twice?

**Review by rowleya — APPROVED** [review]
Looks fine - just a minor question to make sure before merging!

File: docs/spin1_api/Makefile (hunk 1 above)
Comment (lplana) [reply docs/spin1_api/Makefile:18]: Yes .Latex sometimes cannot resolve all references in the first pass. The second call gets the reference right.

Sorry to bother you with this nuisance, but I cannot do the review myself.

Comment (rowleya) [reply docs/spin1_api/Makefile:18]: No probs, just wanted to check!

**Pull request merged**